templates/base.html

```diff
@@ -396,6 +395,9 @@
       {% if "links.html"|template_exists %} {% include "links.html" %}
       {% else %}
       <div class="column col-xs-16 text-right">
+        <a href="https://hikaya.io" class="text-muted"><b>Hikaya</b></a> |
+        <a href="https://help.hikaya.io" class="text-muted"><b>Documentation</b></a>
+        <a href="https://github.com/hikaya-io/Activity-CE/blob/master/LICENSE" class="text-muted"><b>License</b></a>
```

Comment:
Nice work @calixo888, Could you please remove this footer links since they are duplicates of existing ones

Reply:
Done! Mind checking this PR again?